frontend/src/utils/broken-link-disabler.js

```diff
@@ -0,0 +1,11 @@
+const mixin = {
+  methods: {
+    isBrokenLink(link) {
+      return link === undefined;
+    },
+  },
+  data: {
+    disabledLinkMessage: 'This remote link is unsupported',
+  },
+};
+export default mixin;
```

Comment:
We can just merge this with the first line, so just an `export default { ... }` instead of declaring a constant and exporting that constant.